pkg/config/testdata/HoneycombExporter_output_collector_config.yaml

```
@@ -0,0 +1,10 @@
+API_Key: ${HONEYCOMB_TRACES_APIKEY}
+API_URL: https://api.honeycomb.io
```

Naming is always fun. There's 3 different forms here; `API_Key`, `AcessKeys` and `sendKey` :|

```suggestion
ApiKey: ${HONEYCOMB_TRACES_APIKEY}
ApiUrl: https://api.honeycomb.io
```